Trappist/src/Promact.Trappist.Test/ClearDatabase.cs

```diff
@@ -2,10 +2,11 @@
 using Promact.Trappist.DomainModel.DbContext;
 using Promact.Trappist.DomainModel.Enum;
 using Promact.Trappist.DomainModel.Models.Question;
+using System.Linq;
 
 namespace Promact.Trappist.Test
 {
-    public class ClearDatabase
+    public static class ClearDatabase
```

Comment:
the class was made static because the SonarQube check reported a issue and suggested it to change to static.